### DISCUSSION TITLE
Extract Identity Providers from realm json field

### DISCUSSION THREAD

**Discussion by vramik** [keycloak/keycloak#12005] [Archive]
Some users have setup where number identity providers is high. e.g 4k+ (See https://github.com/keycloak/keycloak/discussions/8608).

With such a number of identity providers json field in realm jpa map store implementation would be large and it also seems that performance-wise it might be useful to extract it.

**Comment by ahus1** [comment]
+1 for this. Still it would only be useful if the UI wouldn't fetch this information and we would add pagination as described in the referenced issue. Even if we don't update the UI, at least the REST endpoints should be updated.

Looking at the API changes necessary, I'd rather postpone it :-(
